BGL/include/CGAL/boost/graph/iterator.h

```diff
@@ -980,6 +980,12 @@ faces_around_face(typename boost::graph_traits<Graph>::halfedge_descriptor h, co
   return make_range(I(h,g), I(h,g,1));
 }
 
+/**
+ * \ingroup PkgBGLIterators
+ * A bidirectional circulator with value type `boost::graph_traits<Graph>::%vertex_descriptor` over all vertices incident to the same face or border.
+ * \tparam Graph must be a model of the concept `HalfedgeGraph`
+ * \cgalModels `BidirectionalIterator`
+ */
 template <typename Graph>
 class Vertex_around_face_circulator
```

Comment:
This documentation of `Vertex_around_face_circulator` is useless because it does not explain how to construct such a circulator. I will merge the PR anyway.

It seems to be a common curve of all circulators and iterators in the BGL: only "model of" and the `value_type` are documented. That is fine for a type that is the return type of a function, because in that case the function is the creator, but `Vertex_around_face_circulator` is not used or produced by any function of the CGAL BGL.

Reply:
I had added it because it is mentioned in the [User Manual of `Surface_mesh`.](https://doc.cgal.org/latest/Surface_mesh/index.html#sectionSurfaceMesh_circulators)